tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -557,6 +559,27 @@ public:
    }
 };
 
+/// The field for a class representing a collection of elements via `TVirtualCollectionProxy`.
+/// Objects of such type behave as collections that can be accessed through the corresponding member functions in
+/// `TVirtualCollectionProxy`. At a bare minimum, the user is required to provide an implementation for the following
+/// functions in `TVirtualCollectionProxy`: `HasPointers()`, `GetProperties()`, `GetValueClass()`, `GetType()`,
+/// `PushProxy()`, `PopProxy()`, `GetFunctionCreateIterators()`, `GetFunctionNext()`, and
+/// `GetFunctionDeleteTwoIterators()`.
+///
+/// The collection proxy for a given class can be set via `TClass::CopyCollectionProxy()`.
```

Comment:
Perhaps we need to update this comment; it looks identical to the class documentation for `RProxiedCollectionField`.

Reply:
Indeed, user-defined proxied collections should be allowed given that the required functions are implemented (in fact, that's why the documentation is quite extensive on the RNTuple side).